client/src/components/SandwichShow.js

```diff
@@ -47,8 +94,10 @@ const SandwichShow = (props) => {
       <h2>{sandwich.name}</h2>
       <h3>Restaurant: {sandwich.restaurant}</h3>
       {showDescription}
-      <h4>Reviews:</h4>
+      <h4>Reviews</h4>
       {reviewList}
+      <p onClick={newReview}>Add Review?</p>
```

Comment:
Remove question mark here. Its implited enough and we try to minimize punctuation on intractable elements